src/app/portal/edit-inventory/edit-inventory.component.html

```diff
@@ -63,13 +63,24 @@ <h1 class="tableHeading">Team Inventory</h1>
           >
             Return
           </button>
-          <button
-            *ngIf="item.isBorrowable !== true"
-            (click)="useItemModal(item)"
-            class="btn btn-warning actionButton"
-          >
-            Use 1
-          </button>
+          <div class="buttonContainer">
+            <button
+              *ngIf="item.isBorrowable !== true"
+              (click)="useItemModal(item)"
+              class="btn btn-warning actionButton"
+              style="margin-right: 1rem"
+            >
+              Use 1
+            </button>
+
+            <button
+              *ngIf="item.isBorrowable !== true"
+              (click)="openModal(item)"
```

Comment:
Can we use a more descriptive name for the modal here?

Reply:
Ideen is talking about the name openModal. He says to rename it to openMultipleItemModal.